app/src/main/java/lt/vilnius/tvarkau/API/GetNewProblemParams.java

```diff
@@ -0,0 +1,32 @@
+package lt.vilnius.tvarkau.API;
+
+import android.support.annotation.Nullable;
+
+public class GetNewProblemParams {
+
+    private String session_id;
+    private String description;
+    private String type;
+    private String address;
+    private double x;
+    private double y;
+    private String[] photo;
+    private String email;
+    private String phone;
+    private String message_description;
```

Comment:
Also, there's `setFieldNamingPolicy`, which might be convenient http://stackoverflow.com/a/11810183/517381